wolftpm/tpm2.h

```diff
@@ -2754,6 +2757,26 @@ WOLFTPM_API TPM_RC TPM2_NV_Certify(NV_Certify_In* in, NV_Certify_Out* out);
 
 /* Vendor Specific API's */
 #if defined(WOLFTPM_ST33) || defined(WOLFTPM_AUTODETECT)
+
```

Comment:
Why did this section move up and not near `TPMI_GPIO_NAME`?

Reply:
I moved it up, so both vendor specific sections have the same coding style.  Please see the Nuvoton part below. I could modify this, of course.